test/chainsaw/testmetrics/dynatrace-provider/00-install.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: metrics.keptn.sh/v1beta1
+kind: KeptnMetric
+metadata:
+  name: podtatometric
+  namespace: keptn-system
+spec:
+  provider:
+    name: "dynatrace"
+  query: "query"
+  fetchIntervalSeconds: 5
+
+---
+apiVersion: metrics.keptn.sh/v1beta1
+kind: KeptnMetricsProvider
+metadata:
+  name: dynatrace
+  namespace: keptn-system
+spec:
+  secretKeyRef:
+    key: DT_TOKEN
+    name: dynatrace
+  type: dynatrace
+  targetServer:  (join('.', ['http://mockserver', $namespace, 'svc.cluster.local:1080']))
```

Comment:
```suggestion
  targetServer:  (join('.', ['http://mockserver', $namespace, 'svc.cluster.local:1080']))
  
```

Reply:
@RealAnna What's the change here? They are the same

Reply:
see my other comment